src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5528,4 +5528,10 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_OutVariableCannotBeByRef" xml:space="preserve">
     <value>An out variable cannot be declared as a ref local</value>
   </data>
+  <data name="WRN_NullabilityMismatchInConstraints" xml:space="preserve">
+    <value>Nullability in constraints for type parameter '{0}' of method '{1}' doesn't match the constraints for type parameter '{2}' of interface method '{3}'. Consider using an explicit interface implementation instead.</value>
```

Comment:
>of method [](start = 63, length = 9)

I'm surprised this message is dedicated to methods. What about generic types or interfaces?

Reply:
> I'm surprised this message is dedicated to methods. What about generic types or interfaces?

It is reported in a situation that is specific to methods

---
In reply to: [213846141](https://github.com/dotnet/roslyn/pull/29586#discussion_r213846141) [](ancestors = 213846141)